crates/weaver_resolved_schema/src/registry.rs

```diff
@@ -126,6 +126,10 @@ pub struct Group {
     /// The readable name for attribute groups used when generating registry tables.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub display_name: Option<String>,
+    // /// The body of the event.
+    // /// This fields is only used for event groups.
+    // #[serde(skip_serializing_if = "Option::is_none")]
+    // pub body: Option<AnyValue>,
```

Comment:
Why is this commented out?

Reply:
In this PR as it's intent was to "just" show the AnyValue, this was showing "how" it is used in the Event PR (prior to me updating the Event PR)